Core1Platform/HelpAndSupport/02_CreateTicket.md

```diff
@@ -0,0 +1,51 @@
+# Create ticket
```

Comment:
H1 und H2 sollten nicht gleich heißen. Kann man hier einen allgemeineren Titel verwenden, z.B. Support function? Oder in H2 was anderes verwenden?